src/mount_media.c

```diff
@@ -40,6 +40,7 @@
 #include <fcntl.h>
 #include <libdevinfo.h>
 #include <limits.h>
+#include <fts.h>
```

Comment:
As per the CDDL (file licence) you are free to add a second `Copyright` or `Portions Copyright` line to the start of this file if you wish.

Reply:
Thank you for the reminder. It is just a small contribution this time, so maybe next time.